lfc-scheduler/hack/boilerplate/boilerplate.generatego.txt

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Is this file really needed?

Reply:
I will spend some time cleaning up the hack folder 👍